front/src/Phaser/Game/GameMap.ts

```diff
@@ -130,6 +130,10 @@ export class GameMap {
         return this.map;
     }
 
+    public getTilesetProperties(): { [tile_index: number]: Array<ITiledMapLayerProperty> } {
```

Comment:
You could instead write a function that returns a given Array<ITiledMapLayerProperty> based on the tile index:

```
public getTileProperties(tileIndex: number): Array<ITiledMapLayerProperty> {
```

Reply:
already exists : over here https://github.com/thecodingmachine/workadventure/pull/1223